includes/services.inc.php

```diff
@@ -48,45 +18,25 @@ function add_service($device, $type, $desc, $ip = '', $param = '', $ignore = 0,
         $ip = $device->pollerTarget();
     }
 
-    $insert = ['device_id' => $device->device_id, 'service_ip' => $ip, 'service_type' => $type, 'service_changed' => ['UNIX_TIMESTAMP(NOW())'], 'service_desc' => $desc, 'service_param' => $param, 'service_ignore' => $ignore, 'service_status' => 3, 'service_message' => 'Service not yet checked', 'service_ds' => '{}', 'service_disabled' => $disabled, 'service_template_id' => $template_id, 'service_name' => $name];
+    $insert = ['device_id' => $device->device_id, 'service_ip' => $ip, 'service_type' => $type, 'service_changed' => DB::raw('UNIX_TIMESTAMP(NOW())'), 'service_desc' => $desc, 'service_param' => $param, 'service_ignore' => $ignore, 'service_status' => 3, 'service_message' => 'Service not yet checked', 'service_ds' => '{}', 'service_disabled' => $disabled, 'service_template_id' => $template_id, 'service_name' => $name];
```

Comment:
To avoid DB::raw, you can just send a  timestamp with `time()`.

Alternatively, set the service_changed as the updated_at column in the model and use that:
```
    public const UPDATED_AT = 'service_changed';
    public const CREATED_AT = null;
```
(remove timestamps = false)